pom.xml

```diff
@@ -461,6 +461,7 @@
     <version.mariadb-java-client>1.3.4</version.mariadb-java-client>
 
     <version.ch.obermuhlner>2.0.1</version.ch.obermuhlner>
+    <version.ch.obermuhlner.js>2.3.0</version.ch.obermuhlner.js>
```

Comment:
where does the name `ch.obermuhlner.js` come from please? I do not see similar string in https://github.com/treblereel/gwt-bigmath

Reply:
Good point. I think the GAV changed at one point.